test/system/deleted-files.spec.ts

```diff
@@ -91,6 +116,31 @@ describe("Deleted Folders", () => {
     expect(pluginResult.scanResults.length).toEqual(1);
   });
 
+  /*
+    The following refers to the case where in several layers the exact same packages are added in the same exact path.
+    The image in the example below was built with the following Dockerfile:
+
+    FROM alpine:latest
+    WORKDIR /app
+    RUN mkdir -p /app/tmp/jar_folder
+    RUN mkdir jar_folder && wget https://repo1.maven.org/maven2/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar && mv commons-collections-3.2.1.jar jar_folder
+    RUN wget https://repo1.maven.org/maven2/commons-collections/commons-collections/3.2.1/commons-collections-3.2.1.jar -P /app/tmp/jar_folder
+    RUN mv /app/tmp/jar_folder/commons-collections-3.2.1.jar /app/jar_folder/
+    RUN rm -rf /app/tmp/jar_folder
+    RUN rm -rf /app/jar_folder/*
+  */
+  it("should exclude removed packages added multiple times before a removal", async () => {
```

Comment:
It's not very clear to me at first glance what the difference between this test and the previous text is.

What is the difference between `files` and `packages`?